.test-infra/jenkins/job_LoadTests_Java.groovy

```diff
@@ -0,0 +1,85 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import LoadTestsBuilder as loadTestsBuilder
+import CommonJobProperties as commonJobProperties
+
+
+def testsConfigurations = [
+        [
+                jobName           : 'beam_Java_LoadTests_GroupByKey_Direct_Small',
+                jobDescription    : 'Runs GroupByKey load tests on direct runner small records 10b',
+                itClass           : 'org.apache.beam.sdk.loadtests.GroupByKeyLoadTest',
+                prCommitStatusName: 'Java GroupByKey Small Java Load Test Direct',
+                prTriggerPhase    : 'Run GroupByKey Small Java Load Test Direct',
+                runner            : loadTestsBuilder.Runner.DIRECT,
+                jobProperties     : [
+                        publishToBigQuery: true,
+                        bigQueryDataset  : 'load_test_PRs',
```

Comment:
Can you make the same trick for load tests that is done for Nexmark jobs, i.e. choose the bigQueryDataset based on the `TriggeringContext`? This will become super useful when we add periodic Jenkins post-commit load tests - their result will need to land in a separate dataset.

More specifically i mean: `NexmarkBuilder.determineBigQueryDataset()` method

Reply:
Ok, I'll add it whet it will be necessary.